web/src/views/Settings/TwoFactorAuthentication/CredentialItem.tsx

```diff
@@ -6,7 +6,8 @@ import IconButton from "@mui/material/IconButton";
 import Grid from "@mui/material/Unstable_Grid2/Grid2";
 import { useTranslation } from "react-i18next";
 
-import { FormatDateHumanReadable } from "@i18n/formats";
+import { useRelativeTime } from "@hooks/RelativeTimeString";
+// import { FormatDateHumanReadable } from "@i18n/formats";
```

Comment:
```suggestion
// import { FormatDateHumanReadable } from "@i18n/formats";
```

Reply:
Used [Intl.RelativeTimeFormat()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/RelativeTimeFormat) to properly translate the time strings.

It should be noted that without including `i18next.languages` as the list of default locales for `Intl.RelativeTimeFormat()`, RelativeTimeFormat will use the browser's language settings and ignore any language preferences set/used by react-i18next. Which can lead to the issue where everything is in the correct language except the string returned by RelativeTimeFormat.